docs/cmd/kn_source_apiserver_create.md

```diff
@@ -0,0 +1,43 @@
+## kn source apiserver create
+
+Create a Kubernetes API Server event source
+
+### Synopsis
+
+Create a Kubernetes API Server event source
+
+```
+kn source apiserver create NAME --resources RESOURCE --service-account ACCOUNTNAME --sink SINK --mode MODE [flags]
+```
+
+### Examples
+
+```
+
+  # Create a Kubernetes API Server event source 'k8sevents' which consumes Kubernetes events and send them to service 'mysvc' as a cloudevent
+  kn source apiserver create k8sevents --resources Event:v1:true --service-account myaccountname --sink svc:mysvc
+```
+
+### Options
+
+```
+  -h, --help                     help for create
+      --mode string              'Ref' or 'Resource', the receive adapter controller runs under. (default "Ref")
+  -n, --namespace string         Specify the namespace to operate in.
+      --resources string         comma seperate Kind:APIVersion:isController list, e.g. Event:v1:true,Pod:v2:false
```

Comment:
Please use a realistic example and maybe avoid the comma separated approach. See my comments below please.

Reply:
What does the boolean `controller` flag mean? Can we add this here into the documentation? Also please add which parts are mandatory and which are optional.

If this is too long then add the explanation outside of the options to the top-level command doc.

Reply:
done